package.json

```diff
@@ -8,6 +8,9 @@
     "node": ">=12"
   },
   "scripts": {
+    "build": "npm run build:gitignore && npm run build:ts",
```

Comment:
can you also add `prepublishOnly` hook for this?

Reply:
Done! Added `"prepublishOnly": "npm run build"`.